src/components/barters-table/index.js

```diff
@@ -283,7 +286,7 @@ function BartersTable(props) {
                         sellTo: t('Flea Market'),
                         name: barterRow.rewardItems[0].item.name,
                         value: barterRow.rewardItems[0].item[priceToUse],
-                        source: barterRow.source,
```

Comment:
@Razzmatazzz would this cause any known issues?

Reply:
Not sure, tbh. The website is still largely a mystery to me.

Reply:
I just tested the site all that I could and see no issues with this, I will deploy and let this PR sit over night